underscore.js

```diff
@@ -647,6 +647,23 @@
     return range;
   };
 
+  // Split an **array** into several arrays containing **count** or less elements
+  // of initial array
+  _.chunk = function(array, count) {
+    count = count || 1;
```

Comment:
A shallow clone isn't returned in lodash. It returns the array in chunks of 1. I think returning a shallow clone would be even worse.